src/Compiler/CompilerPasses.cpp

```diff
@@ -132,10 +145,6 @@ void addONNXToKrnlPasses(mlir::PassManager &pm, int optLevel, bool enableCSE,
                    << ONNXOpsStatFormat << "\"\n";
     }
   }
-  // Add instrumentation for Onnx Ops
-  if (maccel.empty() && instrumentStage == Onnx)
-    pm.addNestedPass<func::FuncOp>(onnx_mlir::createInstrumentPass(
-        instrumentOps, instrumentControlBits.getBits()));
```

Comment:
Move this inside `addONNXToMLIRPasses` so that it works in combination with `--EmitONNXIR`